save-agent/src/nativeMain/kotlin/org/cqfn/save/agent/SaveAgent.kt

```diff
@@ -125,11 +127,15 @@ class SaveAgent(private val config: AgentConfiguration,
                     }
                 state.value = AgentState.FINISHED
             }
-            else -> state.value = AgentState.CLI_FAILED
+            else -> {
+                println("SAVE has exited abnormally with status ${executionResult.code}")
```

Comment:
We don't use logger here yet, because it's K/N and we didn't find a suitable logger. Speaking about files, process's stdout will be stored as docker logs, so files are not needed here